### ISSUE TITLE
high cfg scale，low image diverse

### ISSUE THREAD

**Issue by wytcsuch** [facebookresearch/DiT#96]
Thank you very much for your excellent work. We would like to ask two questions:
（1）We observed that when the cfg parameter is increased, the quality of the generated image is significantly improved, but it seems that the diversity of the image decreases also,why? 
（2）In addition, the model tends to generate images that appear more frequently in the training data, but we found that the loss of these images during training is not the lowest, which seems to contradict the theoretical logic, because theoretically, the model should overfit these images and have a lower loss
